search/task_helpers_test.py

```diff
@@ -576,3 +586,51 @@ def test_update_bootcamp(mocker):
     bootcamp_id = 345
     update_bootcamp(bootcamp_id)
     patched.delay.assert_called_once_with(bootcamp_id)
+
+
+@pytest.mark.django_db
+def test_upsert_podcast(mocker):
+    """
+    Tests that upsert_podcast calls update_field_values_by_query with the right parameters
+    """
+    patched_task = mocker.patch("search.tasks.upsert_podcast")
+    podcast = PodcastFactory.create()
+    upsert_podcast(podcast.id)
+    patched_task.delay.assert_called_once_with(podcast.id)
+
+
+@pytest.mark.django_db
+def test_upsert_podcast_episode(mocker):
+    """
+    Tests that upsert_podcast calls update_field_values_by_query with the right parameters
```

Comment:
`Tests that upsert_podcast_episode calls search.tasks.upsert_podcast_episode....`